modal-js/src/app.ts

```diff
@@ -53,6 +53,9 @@ export type SandboxCreateOptions = {
   /** Timeout of the Sandbox container, defaults to 10 minutes. */
   timeout?: number;
 
+  /** The amount of time in seconds that a sandbox can be idle before being terminated. */
```

Comment:
Do we use seconds or milliseconds for the JS SDK? This comment says seconds, but the example uses ms.

Reply:
Good catch, it should be milliseconds! Fixed now.